worker/worker.go

```diff
@@ -558,6 +590,9 @@ func (w *worker) slabsMigrateHandler(jc jape.Context) {
 		return
 	}
 
+	// attach gouging checker to the context
+	ctx := WithGougingChecker(jc.Request.Context(), mp.GougingParams)
```

Comment:
(nit) would be cool if this was some sort of middleware that simply extends all request contexts with gouging params by default.

Reply:
not every worker route needs gouging checks and decorating the request object with a context requires a call to the `bus` to fetch things like the redundancy settings, I don't mind making a middleware for it but wouldn't we want to avoid the overhead if we can?